clang/test/Driver/sycl-offload.cpp

```diff
@@ -14,3 +14,10 @@
 //  OFFLOAD-AVX-NEXT: clang{{.*}} "-cc1" {{.*}}
 //  OFFLOAD-AVX-NEXT-SAME: "-fsycl-is-host"
 //  OFFLOAD-AVX-NEXT-SAME: "-target-cpu" "[[HOST_CPU_NAME]]" "-target-feature" "+avx"
+
+/// Check that the needed -fsycl -fsycl-is-device and -fsycl-is-host options
+/// are passed to all of the needed steps with and without preprocessing
```

Comment:
I'm not sure it's the best way to reformulate this, but the issue I'm having with the sentence can be described by this suggestion:
```suggestion
/// are passed to all of the needed steps, whether preprocessing is the last phase or not
```

Reply:
Updated the comment, and added a couple additional passes to match.